make.ps1

```diff
@@ -258,14 +258,14 @@ if ($command.Length -gt 1)
 
 switch ($execute)
 {
-	"all" { All-Command }
-	"dependencies" { Dependencies-Command }
-	"version" { Version-Command }
-	"clean" { Clean-Command }
-	"test" { Test-Command }
-	"check" { Check-Command }
-	"check-scripts" { Check-Scripts-Command }
-	"docs" { Docs-Command }
+	{"all",           "a"  -contains $_} { All-Command }
+	{"dependencies",  "d"  -contains $_} { Dependencies-Command }
+	{"version",       "v"  -contains $_} { Version-Command }
+	{"clean",         "c"  -contains $_} { Clean-Command }
+	{"test",          "t"  -contains $_} { Test-Command }
+	{"check",         "ck" -contains $_} { Check-Command }
+	{"check-scripts", "cs" -contains $_} { Check-Scripts-Command }
+	{"docs",          "d"  -contains $_} { Docs-Command }
```

Comment:
`docs` and `dependencies` both have `d` as the shorthand. I would just remove the shortcut for `docs` - it's short enough.

Reply:
Done. Otherwise we could give it 'o' as a shortcut